src/main/java/com/ipiecoles/java/java350/model/Employe.java

```diff
@@ -55,16 +60,47 @@ public Integer getNbRtt(){
         return getNbRtt(LocalDate.now());
     }
 
+    /**
+     * Méthode qui retourne le nombre de jours de RTT pour une année donnée
+     * @param d : la date à considérer
+     * @return : le nombre de jours de RTT trouvés sous forme d'integer
+     */
     public Integer getNbRtt(LocalDate d){
-        int i1 = d.isLeapYear() ? 365 : 366;
-        int var = 104;
+        int nbJoursParAn = d.isLeapYear() ? 365 : 366;
```

Comment:
La fameuse erreur que j'ai introduite...
Ca risque de bousculer tes tests du coup
```suggestion
        int nbJoursParAn = d.isLeapYear() ? 366 : 365;
```